### ISSUE TITLE
default config but not run。error msg: couldn't find the plugin "eslint-plugin-jsx-a11y".

### ISSUE THREAD

**Issue by 2012shenghua** [eslint/eslint#13495]
**Tell us about your environment**

* **ESLint Version:**
* **Node Version:**
* **npm Version:**

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**

**Are you willing to submit a pull request to fix this bug?**

**Comment by 2012shenghua** [comment]
Oops! Something went wrong! :(

ESLint: 7.4.0

ESLint couldn't find the plugin "eslint-plugin-jsx-a11y".

(The package "eslint-plugin-jsx-a11y" was not found when loaded as a Node module from the directory "/Users/hua/pdd/code/writing of my/puppeteer-demo".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

**Comment by 2012shenghua** [comment]
i use cmd run ./node_modules/.bin/eslint --init . then produce default config.

 ESLint version：7.4.0
.eslint.js
{
  "name": "puppeteer-project",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "nodemon index.js --exec ts-node",
    "lint": "eslint src"
  },
  "author": "",
  "description": "",
  "dependencies": {
    "puppeteer": "^3.1.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.10.5",
    "@babel/core": "^7.10.5",
    "@babel/node": "^7.10.5",
    "@babel/preset-env": "^7.10.4",
    "@babel/register": "^7.10.5",
    "@types/node": "^14.0.23",
    "@typescript-eslint/eslint-plugin": "^3.6.1",
    "@typescript-eslint/parser": "^3.6.1",
    "eslint": "^7.4.0",
    "eslint-config-airbnb-typescript": "^8.0.2",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-react": "^7.20.3",
    "ts-node": "^8.10.2",
    "typescript": "^3.9.6"
  }
}

**Comment by anikethsaha** [comment]
@2012shenghua 
Did you select to not install the dependencies in `init` command?

I can see there is an issue when we select not to install dependencies, it still lints the file and throws the errors as it is not being installed

**Comment by anikethsaha** [comment]
@2012shenghua can you share your eslint config. also, did you select not to install the dependencies when using `init` command (the last question) ?
